tests/unit/src/Graze/Monolog/Handler/DynamoDbEventHandlerTest.php

```diff
@@ -42,20 +42,19 @@ public function testHandle()
     {
         $record = $this->getRecord();
         $formatter = $this->getMock('Monolog\Formatter\FormatterInterface');
-        $formatted = array('foo' => 1, 'bar' => 2);
+        $raw = ['foo' => 1, 'bar' => 2];
+        $formatted = [
+            'foo' => ['N' => '1'],
+            'bar' => ['N' => '2']
+        ];
```

Comment:
The changes in this file are because `DynamoDbHandler` from `monolog` does different things based on whether the SDK version is 2 or 3.

https://github.com/Seldaek/monolog/blob/master/src/Monolog/Handler/DynamoDbHandler.php#L77